src/EA.Iws.Web.Tests.Unit/Controllers/NotificationApplication/BaseWasteCodeControllerTests.cs

```diff
@@ -1,14 +1,16 @@
 ﻿namespace EA.Iws.Web.Tests.Unit.Controllers.NotificationApplication
 {
-    using System;
```

Comment:
Would you mind updating your Visual Studio settings so that System directives are placed first by default please? I wouldn't worry about these changes for now but just something to keep the code consistent. 

To update your settings you'd have to "_Navigate to Tools --> Options. In the left hand menu pane, expand Text Editor --> C# --> Advanced, and ensure the "Place 'System' directives first when sorting usings" option is selected._"